.gitignore

```diff
@@ -25,3 +25,4 @@ _site
 yarn-error.log
 yarn-debug.log
 npm-debug.log
+.idea
```

Comment:
Is this addition intentional/necessary?

Reply:
`.idea` folder contains project settings for Jetbrains IDEs (Intellij/PhpStrom/WebStorm etc). These IDE pretty popular in webdev, so i thought having it in `.gitingore` could be usefull.
Anyway it's not necessary and could be removed.